src/mmw/js/src/core/templates/modificationPopup.html

```diff
@@ -2,7 +2,11 @@
 <table class='table'>
     <tr>
         <td>Total Area</td>
-        <td class='strong text-right'>{{ area|round(1)|toLocaleString }} {{ units }}</td>
+        <td class='strong text-right'>{{ area|round(1)|toLocaleString(2) }} {{ units }}</td>
+    </tr>
+    <tr>
+        <td>Effective Area</td>
+        <td class='strong text-right'>{{ effectiveArea|round(1)|toLocaleString(2) }} {{ effectiveUnits }}</td>
```

Comment:
Cool reuse of `toLocaleString` for padding. If we're rounding to 1 digit, should we also display only 1 fractional digit? Or round to 2 for visual consistency with the tables? Rounding to 1 and but showing 2 to gives a false impression of accuracy.